test/tests.js

```diff
@@ -6828,7 +6827,7 @@ describe("Tables", function () {
                 [c3, c4]
             ]);
             var result = table._iterateLayout(500, 500);
-            verifyLayoutResult(result, [0, 0], [220, 220], [480, 20], [50, 10], true, false, "");
+            verifyLayoutResult(result, [5, 5], [225, 225], [490, 0], [50, 0], false, false, "");
```

Comment:
I have an odd feeling about this. In particular, the first 2 arguments `[5, 5], [255, 255]` represent `colProportionalSpace` and `rowProportionalSpace` respectively. I do not think this is true. 

The way this should be read is: We have 2 components, they share 10 units of column space and 250 units of row space, which are split equally between them. This would be (somewhat) true if we had no fixed component. The fixed component makes the semantics of this very weird